### ISSUE TITLE
2:1

### ISSUE THREAD

**Issue by userkilled** [hyst329/OpenFool#39]
Does the selected background apply to the main screen?  And I have a 2: 1 background on the screen in the squares of 16: 9, and the remaining sides are green.
![Screenshot_2019-08-23-05-22-32-999_ru hyst329 openfool](https://user-images.githubusercontent.com/39238592/63562398-23d4de80-c566-11e9-9d54-001b1e77f955.png)

**Comment by hyst329** [comment]
Yes, it's a known issue due to some hardcoded values. "Dehardcoding" it is work in progress, it will be fixed in the upcoming 0.3.0 release
